frontend/src/api/questionBankApi.ts

```diff
@@ -0,0 +1,10 @@
+import { IAuthContext } from '../interfaces';
+import { backendServicesPaths } from '../utils/constants';
+import { generateAuthInterceptor } from './authUtils';
+
+// TODO: remove this API call
+// This API shows how to use the auth interceptor
+export async function pingBackend(authContext: IAuthContext) {
+  const { protectedRequests, protectedRequestConfig } = generateAuthInterceptor(authContext);
+  return protectedRequests.get(backendServicesPaths.question.pingAuth, protectedRequestConfig);
+}
```

Comment:
This feels clunky, and leaks abstraction. Can we hide the implementation details of authenticated and unauthenticated endpoints?

e.g. `requestBackend(endpoint, useAuthentication)` - and we attach the JWTs iff `useAuthentication = true`